content/05.conclusion.md

```diff
@@ -1,29 +1,29 @@
-Our work utilized document embeddings to fill in these missing links within bioRxiv, suggesting that these embeddings provide an extensive way to establish preprint-published links.
-Furthermore, our analysis suggests that the publication rate for preprints is in fact higher than previously estimated; however, our analysis only accounts for papers that are published open access.
-This means our results should only be considered to raise the lower bound of the total preprint publication fraction; however, the true fraction is likely to be much higher.
+Our work suggests that document embeddings can help fill in missing links within bioRxiv.
+Furthermore, our analysis reveals that the publication rate for preprints is higher than previously estimated, even though our analysis can only account for papers that are published open access.
+Our results raise the lower bound of the total preprint publication fraction; however, the true fraction is necessarily higher. Future work, especially that which aims to assess the fraction of preprints that are eventually published, should account for the possibility of missed annotations.
 Future work, especially that which aims to assess the fraction of preprints that are eventually published, should account for the possibility of missed annotations.
 
-Preprints take variable amount of time to become published and our work sought to provide more insight on a preprints time to publication.
-Our half-life analysis on preprint categories revealed that majority of categories take the same amount of time for preprints to be published.
-Exception towards this trend is the scientific communication and education category, suggesting that some part of the peer review process greatly halts preprints within this category.
-In respect to individual preprints, each substantial change adds an additional 16 days for a preprint to become published; however, an additional 51 days are added in cases where a preprint has to have a completely new version constructed.
+Preprints take variable amount of time to become published, and we examined factors that influence a preprint’s time to publication.
+Our half-life analysis on preprint categories revealed that preprints in most bioRxiv categories take similar amounts of time to be published.
+A clear exception is the scientific communication and education category, which contained preprints that took much longer to publish.
+In respect to individual preprints, each new version adds several weeks to a preprints time to publication, which is roughly aligned with authors making changes after a round of peer review; furthermore, preprints that undergo substantial changes take longer to publish.
+Overall these results illustrate that bioRxiv is a practical resource for obtaining insight into the peer review process.
 Future direction would be to examine the types of changes requested by the peer review process that results in long delay times.

```

Comment:
```suggestion

```
I think that with your new closing sentence, this can be dropped. I could also see a reviewer getting up in arms about it.